langlib/lang.regexp/src/main/ballerina/regexp.bal

```diff
@@ -320,6 +412,14 @@ isolated function getReplacementString(Groups groups, Replacement replacement) r
 # between matches, or after the last match.  If there are no matches, then
 # `[str]` will be returned.
 #
+# ```ballerina
+#var sampleRegex = re `,`;
+#
+#sampleRegex.split("Not Valid").length() ⇒ 0
```

Comment:
Is this correct?

Reply:
This returns the complete string as a string[1]